src/bundle/ui-dev/src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -30,7 +30,7 @@ const SORTKEY_MAP = {
     priority: 'LocationPriority',
 };
 const TABLE_CELL_CLASS = 'c-table-view__cell';
-const TABLE_HEAD_CLASS = `${TABLE_CELL_CLASS} ${TABLE_CELL_CLASS}--head`;
+const TABLE_HEAD_CLASS = `ibexa-table__header-cell ${TABLE_CELL_CLASS} ${TABLE_CELL_CLASS}--head`;
```

Comment:
Personally I would remove `TABLE_CELL_CLASS` const, it makes these four places where it's used more unreadable, but it's only suggestion